extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -383,26 +383,37 @@ class QueueProcessor extends EventEmitter {
     }
 
     scheduleResume(date) {
+        function triggerResume() {
+            this._updateZkStateNode('scheduledResume', null, err => {
```

Comment:
This node name `scheduledResume` is the same one which was mis-spelt, correct? Why not use a constant 
`const RESUME_NODE = 'scheduledResume';`